data/templates/install-coreos.sh

```diff
@@ -2,4 +2,5 @@
 set -e
 curl -O http://<%=server%>:<%=port%>/api/current/templates/pxe-cloud-config.yml
 sudo coreos-install -d <%=installDisk%> -c pxe-cloud-config.yml -b <%=repo%>
+wget --spider http://<%=server%>:<%=port%>/api/current/templates/<%=completionUri%>
```

Comment:
@codenrhoden : @cgx027 just defined a special notification API to replace previous wget or curl. And we are planing to replace all completionUri with the new notification API.

Since you happened to touch these code, so could you please update the code to use the notification API?